.github/conda/meta.yaml

```diff
@@ -40,7 +40,7 @@ requirements:
     - sacremoses
     - regex !=2019.12.17
     - protobuf
-    - tokenizers >=0.10.1,<0.11.0
+    - tokenizers >=0.11.1,!=0.11.3,<0.13
```

Comment:
I wonder if the changes necessary aren't limited to these? It seems like the error originates from `Searching for tokenizers!=0.11.3,<0.13,>=0.11.1`, so I would try it just with this change

Reply:
I did try with just that but I also got [the other errors that I mentioned in the PR description](https://github.com/huggingface/transformers/runs/5853294684?check_suite_focus=true#step:6:2172).